app/packages/spaces/src/components/PanelNotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import { usePluginComponent } from "@fiftyone/plugins";
+import { Typography } from "@mui/material";
+import { PanelProps } from "../types";
+
+export default function PanelNotFound(props: PanelNotFoundPropsType) {
+  const { panelName } = props;
+  const CustomPanelNotFound = usePluginComponent("PanelNotFound")?.component;
+
+  if (CustomPanelNotFound) {
+    return <CustomPanelNotFound {...props} />;
```

Comment:
Seems reasonable from a design POV to allow this to be over-ridden...but I'm not sure on the use case. What is the actual use case for this?

Reply:
The main use case atm is to enable displaying a more appropriate message in teams environment